stories/RangeInput.stories.js

```diff
@@ -14,4 +14,22 @@ storiesOf('RangeInput', module)
   }))
   .add('with min and max', () => ({
     template: `<ais-range-input attribute="price" :min=10 :max=50></ais-range-input>`,
+  }))
+  .add('using the default slot', () => ({
+    template: `
+      <ais-range-input attribute="price">
+        <template slot-scope="{refine, updateMin, updateMax, currentRefinements}">
+          <form  @submit.prevent="refine()" >
+            <label> 
+              <input type="number" :max="this.max"  :placeholder="this.max" :value="currentRefinements && currentRefinements[0]" @change="updateMin($event.currentTarget.value)"/>
```

Comment:
did you change currentRefinements?

Reply:
I'm not sure I get your question. There is no currentRefinements in the connector. This is the values that you get in the slots and that are provided by the connector (start property)